<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginSignin</class>
 <widget class="QWidget" name="LoginSignin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="firstNameL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>46</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="signUpL">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>-5</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(64, 0, 255);
font: 800 17pt &quot;Rockwell Extra Bold&quot;;
</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-5</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(64, 0, 255);
font: 800 17pt &quot;Rockwell Extra Bold&quot;;
</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>92</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Last Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>105</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>138</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>National Code :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nationalCodeLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>151</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ageL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>183</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Age :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>197</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="signUpUsernameL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>232</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Username :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUpUsernameLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>245</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="signUpPasswordL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUpPasswordLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>293</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginPasswordL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>153</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginUsernameLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>101</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
selection-background-color: rgb(83, 83, 83);
color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginPasswordLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>199</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginUsernameL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>55</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Username :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpPB">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="loginsignup.qrc">:/images/loginsignin.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>59</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>86</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The first name field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>132</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The last name field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>178</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The national code field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>224</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The age field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupUsernameEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>272</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The username field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupPasswordEmptyError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The password field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginUsernameEmptyError">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>128</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The username field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginPasswordEmptyError">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>226</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The password field is empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupUsernameInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>272</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The username field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>178</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The national code field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginPasswordInvalidError">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>226</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The password field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginUsernameInvalidError">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>128</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The username field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupPasswordInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The password field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>224</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The age field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>132</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The last name field is invalid</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameInvalidError">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>86</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The first name field is invalid</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>firstNameL</zorder>
  <zorder>signUpL</zorder>
  <zorder>loginL</zorder>
  <zorder>lastNameL</zorder>
  <zorder>lastNameLE</zorder>
  <zorder>nationalCodeL</zorder>
  <zorder>nationalCodeLE</zorder>
  <zorder>ageL</zorder>
  <zorder>ageLE</zorder>
  <zorder>signUpUsernameL</zorder>
  <zorder>signUpUsernameLE</zorder>
  <zorder>signUpPasswordL</zorder>
  <zorder>signUpPasswordLE</zorder>
  <zorder>loginPasswordL</zorder>
  <zorder>loginUsernameLE</zorder>
  <zorder>loginPasswordLE</zorder>
  <zorder>loginUsernameL</zorder>
  <zorder>loginPB</zorder>
  <zorder>signUpPB</zorder>
  <zorder>firstNameLE</zorder>
  <zorder>firstNameEmptyError</zorder>
  <zorder>lastNameEmptyError</zorder>
  <zorder>nationalCodeEmptyError</zorder>
  <zorder>ageEmptyError</zorder>
  <zorder>signupUsernameEmptyError</zorder>
  <zorder>signupPasswordEmptyError</zorder>
  <zorder>loginUsernameEmptyError</zorder>
  <zorder>loginPasswordEmptyError</zorder>
  <zorder>signupUsernameInvalidError</zorder>
  <zorder>nationalCodeInvalidError</zorder>
  <zorder>loginPasswordInvalidError</zorder>
  <zorder>loginUsernameInvalidError</zorder>
  <zorder>signupPasswordInvalidError</zorder>
  <zorder>ageInvalidError</zorder>
  <zorder>lastNameInvalidError</zorder>
  <zorder>firstNameInvalidError</zorder>
 </widget>
 <resources>
  <include location="loginsignup.qrc"/>
 </resources>
 <connections/>
</ui>
